microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -26,7 +26,7 @@
     This test suite runs the gpu test cases.
     """,
 )
-class gpu(TestSuite):
+class gpu(TestSuite):  # noqa
```

Comment:
Why? These ones look indeed bad...

Reply:
Because I was not sure if it's intended to be so..